cmd/common_test.go

```diff
@@ -237,6 +237,7 @@ func TestMapToCISVersion(t *testing.T) {
 		{kubeVersion: "1.19", succeed: true, exp: "cis-1.20"},
 		{kubeVersion: "1.20", succeed: true, exp: "cis-1.20"},
 		{kubeVersion: "1.21", succeed: true, exp: "cis-1.20"},
+		{kubeVersion: "1.21", succeed: true, exp: "cis-1.23"},
```

Comment:
I think unit tests are failing because of it
```suggestion
		{kubeVersion: "1.23", succeed: true, exp: "cis-1.23"},
```

Reply:
oops ...